DQM/L1TMonitor/src/L1TStage2EMTF.cc

```diff
@@ -10,6 +9,8 @@ L1TStage2EMTF::L1TStage2EMTF(const edm::ParameterSet& ps)
       trackToken(consumes<l1t::EMTFTrackCollection>(ps.getParameter<edm::InputTag>("emtfSource"))),
       muonToken(consumes<l1t::RegionalMuonCandBxCollection>(ps.getParameter<edm::InputTag>("emtfSource"))),
```

Comment:
To fix the ProductNotFound error every token should have its own InputTag in order to choose different instance names. They should not use all `emtfSource`.